alg/teca_cartesian_mesh_source.h

```diff
@@ -54,50 +54,137 @@ class teca_cartesian_mesh_source : public teca_algorithm
     TECA_GET_ALGORITHM_PROPERTIES_DESCRIPTION()
     TECA_SET_ALGORITHM_PROPERTIES()
 
-    // set/get the type code for generated coordinates.
-    // default is 32 bit floating point. Use
-    // teca_variant_array_type<NT>::get() to get specific type
-    // codes for C++ POD types NT.
+    /** @anchor coordinate_type_code
```

Comment:
Ooops. I also made major revisions to teca_cartesian_mesh_source class and its documentation. I've added and revised the documentation in addition to the stylistic changes you made. To avoid losing the new text during conflict resolution, please drop these changes.